phira/locales/en-US/rate.ftl

```diff
@@ -1,11 +1,11 @@
 
 rate = Rate
-filter = Filter by rating
+filter = Filter by Rating
 
 cancel = Cancel
 confirm = Confirm
 
-lower-bound = Lower bound
-upper-bound = Upper bound
+lower-bound = Lower-Bound
```

Comment:
idk but "Lower/Upper Bound" is legal for noun. 
Do you want to let it as a adjective?